lib/titanium.js

```diff
@@ -176,7 +176,7 @@ function run(locale) {
 	env.installPath = defaultInstallLocation || env.installPath;
 
 	// initialize the cli processor
-	var cli = new(require('./cli'))({
+	var cli = new (require('./cli'))({
```

Comment:
<!--
  1 failure:  lib/titanium.js l...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: lib/titanium.js;
  Line: 179;
-->
- :no_entry_sign: lib/titanium.js line 179 – Unexpected space after unary word operator 'new'. (space-unary-ops)

Reply:
No build, you are wrong